README.md

```diff
@@ -341,7 +341,7 @@ Additional lists you might find useful:
 ## Security
 *Plugins and information around security, preventing vulnerabilities and protection against XSS and alike.*
 
-- [BryanCrowe/EncryptedType](https://github.com/bcrowe/cakephp-encrypted-type) - Simple solution that enables Encrypted data types to be stored in the database.
+- :strawberry: [BryanCrowe/EncryptedType](https://github.com/bcrowe/cakephp-encrypted-type) - Simple solution that enables Encrypted data types to be stored in the database.
```

Comment:
This plugin doesn't have any Cake 4 compatible release, just a branch. At least a beta release is necessary to earn the strawberry icon.

Reply:
@bcrowe said he was going to do a release.

Reply:
I'll split up the PR.